content/03.categorize.md

```diff
@@ -5,18 +5,8 @@ Once diagnosed with a disease, an individual might be assigned a stage based on
 While these rules are refined over time, the process is evolutionary and ad hoc, potentially impeding the identification of underlying biological mechanisms and their corresponding treatment interventions.
 
 Deep learning methods coupled with a large corpus of patient phenotypes may provide a more data-driven approach to patient categorization.
```

Comment:
Suggest a smoother transition between these two opening sentences. Such as:

Deep learning methods coupled with a large corpus of patient phenotypes may provide a meaningful and more data-driven approach to patient categorization. For example, they may identify new shared mechanisms that would otherwise be obscured due to ad hoc historical definitions of disease.

Reply:
👍